Sources/SourceKitLSP/SourceKitServer.swift

```diff
@@ -1195,30 +1218,23 @@ extension SourceKitServer {
       return .success([])
     }
 
-    let fallbackLocation = useLocalFallback
-      ? [symbol.bestLocalDeclaration].compactMap { $0 }
-      : []
+    let fallback: [(occurrence: SymbolOccurrence?, location: Location)] =
+      useLocalFallback
+        ? (symbol.bestLocalDeclaration.map { [(occurrence: nil, location: $0)] } ?? [])
+        : []
```

Comment:
Just wondering whether this would read better as follows. Maybe it’s just my mind that’s weird but I find thinking about `map` on `Optional` harder to reason about than `if` statements. I’ll leave it up to you what you like better.
```suggestion
    let fallback: [(occurrence: SymbolOccurrence?, location: Location)]
    if useLocalFallback, let bestLocalDeclaration = symbol.bestLocalDeclaration {
      fallback = [(occurrence: nil, location: $0)]
    } else {
      fallback = []
    }
```

Reply:
I have no strong preference here, let's go with the if-version then :+1: